Sources/MediaEditor.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
 */
 open class MediaEditor: UINavigationController {
     /// The capabilities are displayed in the Media Editor. You can add your own capabilities here.
-    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorFilters.self, MediaEditorCropZoomRotate.self]
+    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorFiltersViewController.self, MediaEditorCropZoomRotate.self]
```

Comment:
I find it a bit odd that one of the capabilities has the 'ViewController' in the name and the other doesn't. Thoughts?

Reply:
Renamed in 65677df328cdd048fd6dfa30b02a939db6e7c31d